sdk/python/kfp/compiler/compiler.py

```diff
@@ -678,6 +678,9 @@ def _create_dag_templates(self, pipeline, op_transformers=None, op_to_templates_
                             launcher_image=self._launcher_image)
       templates.extend(op_to_templates_handler(op))
 
+      if hasattr(op, 'custom_job_spec'):
+        raise AttributeError('Cannot use `.set_custome_job_spec()` in v1.')
```

Comment:
Instead of raising an error, log a warning, and let the pipeline execute without custom job spec:
`CustomJob spec is not supported yet when running on KFP. The component will execute within the KFP cluster.`

Reply:
Done.